wp/sections/feature.tex

```diff
@@ -1,4 +1,4 @@
-\par As described above, diKTat is very configurable and user-friendly. But these are not all of its advantages and features. Below will be presented and described unusual and important killer-features of diKTat.
+\par As described above, diKTat is configurable and user-friendly. But these are not all of it's advantages and features. Below we will present and describe unusual and important killer-features of diKTat.
 
 \subsection{Configuration file}
 \par
```

Comment:
Is there syntax highlighting for yml in latex? You now have `language=Kotlin` it this block.